deployment/build.debian.armhf

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+#= Debian 10+ arm64 .deb
+
+set -o errexit
+set -o xtrace
+
+# Move to source directory
+pushd ${SOURCE_DIR}
+
+if [[ ${IS_DOCKER} == YES ]]; then
+    # Remove build-dep for dotnet-sdk-3.1, since it's installed manually
+    cp -a debian/control /tmp/control.orig
+    sed -i '/dotnet-sdk-3.1,/d' debian/control
+fi
+
+# Build DEB
+export CONFIG_SITE=/etc/dpkg-cross/cross-config.${ARCH}
+dpkg-buildpackage -us -uc -a armhf --pre-clean --post-clean
```

Comment:
```suggestion
dpkg-buildpackage -us -uc -a ${ARCH} --pre-clean --post-clean
```
?

Reply:
No, it's the opposite - the ARCH being arm* was a bug above.